test/alternator/conftest.py

```diff
@@ -79,6 +80,9 @@ def dynamodb(request):
         # for local runs.
         if request.config.getoption('url') != None:
             local_url = request.config.getoption('url')
+        elif request.config.getoption('host') is not None:
+            # this argument needed for compatibility with PythonSuite without modifying the previous behavior
```

Comment:
What does "PythonSuite" refer to? Is it test.py? test/pylib? Something new you're introducing?

Anyway, "host" is simply not good enough, we need a URL - we need http vs https, we need a port number... So why bother to accept a parameter "host" which isn't good enough?

Reply:
"host" option used by PythonSuite. So we have overlapping options and we need to modify it somewhere. This variant leaves the possibility to use run as previously and use pythonSuite without modification. Migration to the "url" option in PythonSuite will trigger too many changes and it goes against your approach to iterative changes. So this workaround is good enough.

Reply:
PythonSuite is a type of test used in test.py. Currently, alternator have RunSuite type.

Reply:
1. About "host" - I give up. Please just change the comments to clarify that "--host" is a compatibility option that can be used to pick a host but default the scheme to http and port to 8000 - and that "--url" should be preferred to give all of them.

2. "git grep PythonSuite" returned nothing. I see now it's called PythonTestSuite but maybe it's worth saying something like "test.py's PythonTestSuite" to give some context to that name.

Reply:
test.py's PythonTestSuite has two additional parameters: 'ssl' and 'port'. This information will be enough to get the full scheme, but you are insisting on not changing the conftest, so this is a minimal change that we need. We are still couples with information that we are providing in suite.yaml about the alternator port.

Reply:
I guess it does make sense to add a triplet of options, "--ssl", "--port" and "--host" as an _alternative_ to "--url" for compatibility of the parameters in cql-pytest/conftest.py. If you want to do this (and explain in a comment that this is indeed these three options together form an alternative to "--url", I won't be against it. Earlier I didn't understand why you wanted to do this and didn't understand that the framing wasn't about "fixing" anything, it's just about an alternative way to do something for compatibility with cql-pytest.
Or you can leave just the "host" for now if that's enough for now and we can add the rest later - whatever you prefer.